pkg/schema/definitions/converter_test.go

```diff
@@ -0,0 +1,204 @@
+package definitions
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+)
+
+func TestCRDToDefinition(t *testing.T) {
```

Comment:
Some other cases to cover:
- Array of type arrray/object
- Required field
- No items schema in the array
- For an array props.Items.JSONSchemas has some values.